src/win/fs-event.c

```diff
@@ -344,6 +344,15 @@ int uv_fs_event_stop(uv_fs_event_t* handle) {
 }
 
 
+static int file_info_cmp(WCHAR* str, WCHAR* file_name, int file_name_len) {
+  int str_len = wcslen(str);
+  int byte_len = file_name_len / sizeof(WCHAR);
+  int len = str_len > byte_len ? str_len : byte_len;
```

Comment:
Note: Since we only really care about equality, the `_wcsnicmp()` call could be skipped if the lengths aren't equal.